openmetadata-ui/src/main/resources/ui/src/utils/EntityUtils.tsx

```diff
@@ -830,6 +915,7 @@ const getDatabaseOverview = (databaseDetails: Database) => {
       value: <OwnerLabel hasPermission={false} owners={owners} />,
       visible: [DRAWER_NAVIGATION_OPTIONS.explore],
     },
+    ...getCommonOverview({ domain }, false),
```

Comment:
Why we want to hide owner here?

Reply:
For some cases we have an owner option internally in the component based on some conditions, and since to not double show it have added the condition.